hs/database/JsonParser.hs

```diff
@@ -33,57 +33,58 @@ courseDirectory = "../../res/courses/"
 
 -- | A Lecture.
 data Lecture =
-    Lecture { extra      :: Int,
-              section    :: T.Text,
-              cap        :: Int,
-              time_str   :: T.Text,
-              time       :: [[Int]],
```

Comment:
I decided to remove some of the vertical alignment. The `fromJSON` alignment has been kept in, however.